src/components/Members/MembersGrid/MembersGrid.js

```diff
@@ -1,82 +1,154 @@
 
-    this.tryRequire = this.tryRequire.bind(this)
-    this.memberIcon = this.memberIcon.bind(this)
-    this.handleShowModal = this.handleShowModal.bind(this)
-    this.handleHideModal = this.handleHideModal.bind(this)
-    this.numberOfColumns = this.numberOfColumns.bind(this)
-    this.updateNumberOfColumns = this.updateNumberOfColumns.bind(this)
-    this.generateGridList = this.generateGridList.bind(this)
+    this.tryRequire = this.tryRequire.bind(this);
+    this.memberIcon = this.memberIcon.bind(this);
+    this.handleShowModal = this.handleShowModal.bind(this);
+    this.handleHideModal = this.handleHideModal.bind(this);
+    this.numberOfColumns = this.numberOfColumns.bind(this);
+    this.updateNumberOfColumns = this.updateNumberOfColumns.bind(this);
+    this.generateGridList = this.generateGridList.bind(this);
+    this.keyFunction = this.keyFunction.bind(this);
+    this.carouselItem = this.carouselItem.bind(this);
+    this.updateMember = this.updateMember.bind(this);
+    this.changeMemberUIArrows = this.changeMemberUIArrows.bind(this);
 
-    this.members = props.members
+    this.active_members_keys =
+      this.props.active_members.map((member) => (member.id));
+    this.inactive_members_keys =
+      this.props.inactive_members.map((member) => (member.id));
 
     this.state = {
       show_modal: false,
       member: this.props.active_members[0],
+      active: true,
       number_of_columns: this.numberOfColumns(),
-    }
+    };
   }
 
+  /**
+  * Adds key and resizing listeners
+  */
   componentDidMount() {
-    window.addEventListener('resize', this.updateNumberOfColumns)
+    document.addEventListener('keydown', this.keyFunction, false);
+    window.addEventListener('resize', this.updateNumberOfColumns);
   }
 
+  /**
+  * Calculates how many members per row to display on the member's grid.
+  * @return {int} Number of members per row for grid.
+  */
   numberOfColumns() {
-    if (window.innerWidth >= LARGE_WIDTH) return 5
-    if (window.innerWidth >= MEDIUM_WIDTH) return 4
-    if (window.innerWidth >= MOBILE_WIDTH) return 3
-
-    return 2
+    if (window.innerWidth >= LARGE_WIDTH) {
+      return 5;
+    }
+    if (window.innerWidth >= MEDIUM_WIDTH) {
+      return 4;
+    }
+    if (window.innerWidth >= MOBILE_WIDTH) {
+      return 3;
+    }
+    return 2;
   }
 
-  tryRequire(img_path) {
+  /**
+  * Parses path to member image
+  * @param {string} imgPath Path to member's photographs.
+  * @return {path}
+  */
+  tryRequire(imgPath) {
     try {
-      return require(`images/members/${img_path}`)
+      return require('images/members/' + imgPath);
     } catch (err) {
       return placeholder
     }
   }
 
-  handleShowModal(member, event) {
+  /**
+  * Key operations to hide and change modals.
+  * @param {event} event
+  */
+  keyFunction(event) {
+    if (event.keyCode === 27) {
+      this.handleHideModal();
+    }
+    const difference = (event.keyCode === 37)?-1:(event.keyCode === 39)?1:0;
```

Comment:
What those `difference` stands for? Can it be changed to a more descriptive name?
```suggestion
    const difference = (event.keyCode === 37) ? -1 : (event.keyCode === 39) ? 1 : 0;
```

Reply:
I substituted it with "index_difference"